tests/unit/layouts/ModelizerDrawLayout.spec.js

```diff
@@ -142,5 +142,25 @@ describe('Test page component: ModelizerDrawLayout', () => {
 
       expect(Notify.create).not.toHaveBeenCalled();
     });
+
+    it('should arrange a component position and redraw on click', async () => {
```

Comment:
Its in wrong describe, please put it in a new `describe('Test function: arrangeComponentsPosition', () => {})`
outside the `describe('Test function: initView', () => {`

Reply:
done

Reply:
not done, bad correctione see with you by message

Reply:
done